docs/v0.3/en/greptimecloud/integrations/influxdb.md

```diff
@@ -5,7 +5,7 @@ protocol](https://docs.influxdata.com/influxdb/cloud/reference/syntax/line-proto
 ingestion API over http. The API and authentication is compatible with [InfluxDB
 write protocol
 1.x](https://docs.influxdata.com/influxdb/v1.8/guides/write_data/#write-data-using-the-influxdb-api).
-Please refer to [InfluxDB client](https://docs.greptime.com/user-guide/clients/influxdb-line) of GreptimeDB for more information.
```

Comment:
@nicecui so we need to preserve these links also?

Reply:
y, we need to preserve these links